app/serializers/curator/metastreams/administrative_serializer.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Curator
-  class Metastreams::AdministrativeSerializer < CuratorSerializer
+  class Metastreams::AdministrativeSerializer < Curator::Serializers::AbstractSerializer
     schema_as_json root: :administrative do
       attributes :description_standard, :flagged, :harvestable, :destination_site
```

Comment:
When `:flagged` or `:harvestable` are `false`, they are not rendered/serialized -- is that intentional?

Reply:
Yes that is how i designed it. It does not serialize anything that is `blank`, `nil` or `false`. I can make it so it does serialize false values but i would prefer if `nil`, or `blank` values do not get serialized.